apiserver/admin.go

```diff
@@ -300,7 +300,7 @@ func (a *admin) authenticate(ctx context.Context, req params.LoginRequest) (*aut
 		for _, authenticator := range a.srv.loginAuthenticators {
 			var err error
 			authInfo, err = authenticator.AuthenticateLoginRequest(ctx, a.root.serverHost, modelUUID, authParams)
-			if errors.Is(err, errors.NotFound) || errors.Is(err, errors.NotImplemented) {
+			if errors.Is(err, errors.NotFound) || errors.Is(err, errors.NotImplemented) || errors.Is(err, errors.NotSupported) {
```

Comment:
Do we still need NotFound. 
I think NotImplemented and NotSupported should be all we would want (or at least I hope so).
Even NotImplmented seems a little strange. The authenticators could just return NotSupported if there request is something they do not knows how to handle.